tools/qc/fastq_utils/fastq_pre_barcodes.xml

```diff
@@ -0,0 +1,145 @@
+<tool id="fastq_pre_barcodes" name="FASTQ barcodes preprocessor" profile="10" version="conda-package-version+galaxy0">
+    <description>Preprocesses the reads to move the barcodes (UMI, Cell, ...) to the respective readname, optionally discarding reads with bases in the barcode regions below a given threshold.</description>
```

Comment:
Missing the requirements as well (the bioconda package that this will use to run)

Reply:
Done in 51b56dbf19aedfbda7dd8c80d77c0b920da60aa8 but I'm not sure if I referenced the correct version for samtools.

Reply:
I suspect that that will be hard to know, @pinin4fjords might be able to point you to where IRAP is installed on Noah to check the version used. We could in principle try a few runs with this (I suspect most up to date) version and if results are equivalent maybe we keep the newest version. Although maybe for a start, might be better to go if possible with the currently used version in noah.

Reply:
I tried here: https://wwwdev.ebi.ac.uk/gxa/experiments/E-MTAB-2706/Supplementary%20Information but it is not mentioned.

Reply:
irap has samtools samtools 1.9, fastq_utils 0.16.3

Reply:
Changed in 3896d3d2f77c47f9eca79bff42e7df7e12bbfdef, but the test log says it's still using fastq_utils 0.25.1.  Any idea how I might force it to use the correct fastq_utils version?

Reply:
I'm not seeing what you mean in the logs right now, but it may be because that version isn't available in Conda- see https://anaconda.org/bioconda/fastq_utils/files. You could try picking the oldest version available for now, but since we can't easily match versions maybe we should bite the bullet and use the latest. Okay with you @pcm32 ?

Reply:
The html output of the local planemo test that I ran says `fastq_utils 0.25.1` in its report.  Not sure how to view the html here, but maybe they're using the same version.  

According to the fastq_utils repo, these are the dependencies:
```samtools (version 0.1.19) and zlib (http://zlib.net/) version 1.2.11 or latest are required to compile fastq_utils. ... The bam_annotate.sh script requires samtools (version 1.5 or higher).```

Reply:
Changed to latest version in c40fc6aa04058607e98bbd22669a7f354f952361